master/internal/resourcemanagers/provisioner/gcp_config.go

```diff
@@ -225,7 +225,10 @@ func (c GCPClusterConfig) SlotType() device.Type {
 
 // Accelerator returns the GPU accelerator for the instance.
 func (c GCPClusterConfig) Accelerator() string {
-	return c.InstanceType.GPUType
+	return fmt.Sprintf("%d x %s",
+		c.InstanceType.GPUNum,
```

Comment:
aws provisioner doesn't seem to indicate the number of accelerators in Accelerator field either: https://github.com/determined-ai/determined/blob/master/master/internal/resourcemanagers/provisioner/aws_config.go#L243

why is this change being done only for GCP?